CockpitCore/src/main/java/com/polidea/cockpit/core/mapper/YamlToParamMapper.kt

```diff
@@ -48,6 +48,12 @@ internal class YamlToParamMapper {
     private fun createCockpitListType(paramName: String, valueMap: Map<*, *>): CockpitListType<Any> {
         val values = valueMap[MapperConsts.KEY_LIST_VALUES] as? List<*>
                 ?: throw CockpitParseException("$paramName parameter must contain list of elements in `${MapperConsts.KEY_LIST_VALUES}` field")
+
+        val types = values.distinctBy { it!!::class.java }.count()
```

Comment:
We should avoid not-null assertions, otherwise if `it` is null the user will get null pointer exception. What about checking null occurrence and throw more meaningful exception?